core/src/main/scala/kafka/log/LogCleaner.scala

```diff
@@ -91,6 +91,7 @@ import scala.util.control.ControlThrowable
  * @param initialConfig Initial configuration parameters for the cleaner. Actual config may be dynamically updated.
  * @param logDirs The directories where offset checkpoints reside
  * @param logs The pool of logs
+ * @param logDirFailureChannel The channel used to add offline log dirs that may occur when cleaning the log
```

Comment:
I think `occur` should be replaced with `be encountered`. But this seems to be a nitpick.